staff/gustavo-rojas/git/README.md

```diff
@@ -0,0 +1,32 @@
+#To create a repository in the actual directory

```

Comment:
Add an space after # and before first letter to be consistent in all lines. Please validate every step to follow this comment.

Reply:
Done